### PR TITLE
fix: added tear off wrappers to analytics, blotter and tile routes, s…

### PR DESCRIPTION
**Pull request by stewart-adaptive**
…tyle fixes (ARTP-1320 ARTP-1319)

Style fixes to AnalyticsRouteStyle to align top and bottom of panel with other panels.

Added '_ext' to names for torn off windows
check window name in routes to see if we are in a torn off window

TearOff wrappers added for Analytics, Blotter and Live rates

image

### CODE REVIEW

File: src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
+import { inExternalWindow } from './inExternalWindow'
 
-const AnalyticsRouteStyle = styled.div`
+const AnalyticsRouteStyle = styled.div<{ showOpenFinControls : boolean }>`
   /*height offset is needed for openfin controls*/
-  height: calc(100% - 24px);
-  padding: 0 0.625rem 0rem 0.625rem;
-  overflow-x: scroll;
+  height: ${({ showOpenFinControls }) => (showOpenFinControls ? 'calc(100% - 24px)' : '100%')};
+  padding: 0.5rem 0.625rem 0.25rem 0.625rem;
```
Comment (rikoe) [inline src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx:14]: I think the approach here is good, but can we rename `showOpenFinControls` to just `useHeightOffset` or something like that?

We try to keep OpenFin-specific references isolated to the OpenFin module as much as possible.

**Review by rikoe — COMMENTED** [review]
This is awesome @stewart-adaptive! 🎉

File: src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx (hunk 1 above)
Comment (stewart-adaptive) [reply src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx:14]: done

File: src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
-  overflow-x: scroll;
+  height: ${({ useHeightOffset  }) => (useHeightOffset  ? 'calc(100% - 24px)' : '100%')};
+  padding: 0.5rem 0.625rem 0.25rem 0.625rem;
+  overflow-x: auto;
   margin: auto;
 `
-
 const AnalyticsRoute = () => {
+  const analytics = useSelector(analyticsSelector)
+  const isExternalWindow = inExternalWindow
```
Comment (chuck-adaptive) [inline src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx:20]: I noticed this redeclared in all instances where it is used. Any consideration to importing as an alias or renaming the function/file?

**Review by chuck-adaptive — DISMISSED** [review]
👍

File: src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx (hunk 2 above)
Comment (stewart-adaptive) [reply src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx]: done

**Review by chuck-adaptive — APPROVED** [review]

**Pull request merged**